app-dicts/myspell-en/myspell-en-20180901.ebuild

```diff
@@ -0,0 +1,55 @@
+# Copyright 1999-2018 Gentoo Authors
```

Comment:
Why not 2018.11.01? Just because it's from the future?

Reply:
I mean I see a newer version upstream.

Reply:
sorry i still don't understand what you want me to do here

Reply:
Update the pull request to bump to 20181101?

Reply:
Ok sorry, the that's strange, it must have just happened.

Fixed